README.md

```diff
@@ -1,6 +1,23 @@
 ![Release workflow status](https://github.com/JoinMarket-Org/joinmarket-clientserver/actions/workflows/unittests.yml/badge.svg)
+# Table of contents
+---
+* [Table of contents](#table-of-contents)
+* [joinmarket-clientserver](#joinmarket-clientserver)
+* [Wallet features](#wallet-features)
+* [Quickstart - RECOMMENDED INSTALLATION METHOD (Linux and macOS only)](#quickstart---recommended-installation-method-linux-and-macos-only)
+* [More installation guides](#more-installation-guides)
+* [Usage](#usage)
+* [PayJoin](#payjoin)
+* [Joinmarket-Qt](#joinmarket-qt)
+* [Architecture notes](#architecture-notes)
+* [Testing](#testing)
+* [Community](#community)
+* [Support JoinMarket and bitcoin privacy](#support-joinmarket-and-bitcoin-privacy)
+* [Other useful links](#other-useful-links)
+
 
 # joinmarket-clientserver
```

Comment:
On this, and on the name in the ToC: I know it already is called `joinmarket-clientserver` today, since this is automatically generated when you create the README, but that name is kind of a technical name (it's an *implementation* of Joinmarket, but since it is *the* defacto implementation (there are no others in use), and since the entire following paragraph describes *Joinmarket*, not *joinmarket-clientserver*, I think you should change this heading to "Joinmarket".

(Part of what motivates me to make this rather obtuse point is that I greatly regret calling this repo 'joinmarket-clientserver' as it's kind of an awful name!)

Reply:
Updated.